IGI.Lab2/src/CarPark.DAL/Models/ApplicationUser.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Microsoft.AspNetCore.Identity;
+
+namespace CarPark.DAL.Models
+{
+    public class ApplicationUser : IdentityUser 
```

Comment:
Why you don't used IdentityUser directly and create this model?

Reply:
Initially, I wanted to expand the IdentityUser model, then I changed my mind about something))
I forgot to delete it)
But in this case, the program is already ready to expand the User model)

Reply:
I think migrate works only when db exist.
EnsureCreated still should be.

Reply:
fix